test/pytest/mocktransport.py

```diff
@@ -54,15 +54,17 @@ async def check_open(
         channel=None,
         problem=None,
         reply_keys: Optional[JsonObject] = None,
+        absent_keys: 'Iterable[str]' = (),
```

Comment:
With `Iterable` you actually don't need to quote this — but it doesn't hurt, either.

Reply:
I thought we moved typing to quoting, I am kinda confused.